Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -554,10 +560,13 @@ theorem sub_sub (a b c : Ordinal) : a - b - c = a - (b + c) :=
 theorem add_sub_add_cancel (a b c : Ordinal) : a + b - (a + c) = b - c := by
   rw [← sub_sub, add_sub_cancel]
 
-theorem sub_isLimit {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
+theorem isLimit_sub {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
```

Comment:
```suggestion
theorem IsLimit.sub {a b} (l : IsLimit a) (h : b < a) : IsLimit (a - b) :=
```
no?

Reply:
Once I finally get around to replacing `IsLimit` with `IsSuccLimit`, dot notation will break. So I'd rather not introduce it here.

Reply:
Hmm, okay

Reply:
Can you still add the `_left` and `_right` as appropriate?